### DISCUSSION TITLE
Phoenix not register MiningPoolHub

### DISCUSSION THREAD

**Discussion by lukras** [RainbowMiner/RainbowMiner#1722] [Q&A]
Phoenix is ​​unable to register with MPH and a link to NiceHash appears. If you block Phoenix, Trex will do the right thing. If you select MiningPoolHubCoins in the settings, the system does not pay attention to MPH and does not mine profitable ETH on it
![phoenix](https://user-images.githubusercontent.com/30388030/132988777-26d65b98-b89c-44ff-970f-b625b217f3e3.png)

**Comment by RainbowMiner** [comment]
The problem should be solved with the newest release v4.7.4.3
